core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -71,17 +70,13 @@ private static class PrefixProps {
   /**
    * Gets a property value from this configuration.
    *
-   * <p>
-   * Note: this is inefficient, but convenient on occasion. For retrieving multiple properties, use
-   * {@link #getProperties(Map, Predicate)} with a custom filter.
-   *
```

Comment:
The comment that this is inefficient for multiple properties still applies, the advice to use filter maybe should be adjusted to the new methods, but in general prefer getting multiple properties so the "read" to get the underlying configs is minimized.

Reply:
> The comment that this is inefficient for multiple properties still applies

Yeah, that's fair. I can put the comment back into the javadoc for that method.

Reply:
javadoc reverted in 300afea.